components/cards/WhatWeDoCard.jsx

```diff
@@ -3,7 +3,7 @@ import { H4 } from '../ui/Headings';
 
 const WhatWeDo = ({index, title, description, image}) => {
     return (
-        <div className='flex flex-col md:flex md:flex-row md:justify-between  gap-8 md:grid-cols-2 py-8 md:py-14.25 md:mt-12.5'>
+        <div className='flex flex-col md:flex md:flex-row md:justify-between gap-8 md:grid-cols-2 py-8 md:py-14.25 md:mt-12.5'>
             <div className={`md:w-3/4 lg:w-153.5 xl:w-141.25 space-y-4 order-2 ${index % 2 !== 0 ? 'md:order-2 ' : 'md:order-1'}`}>
```

Comment:
Unnecessary white space after `md:order-2` class should be removed.

Reply:
thanks for pointing out, it's resolved!!